conda/tempo-examples.yaml

```diff
@@ -16,3 +18,11 @@ dependencies:
       - xgboost
       - ansible
       - openshift
+      - black==20.8b1
```

Comment:
Do we need this in environment that is just meant to be used by user to run the examples?

Reply:
probably not - the examples just use pytest